<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeSelectorDialog</class>
 <widget class="QDialog" name="ModeSelectorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>437</width>
    <height>245</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayoutAll">
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutNavMode">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labNavMode">
         <property name="text">
          <string>Mode:</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutNavMode">
         <item>
          <spacer name="horizontalSpacerNavMode">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioBtnOnboard">
           <property name="text">
            <string>Onboard</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerOn_Offboard">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QRadioButton" name="radioBtnOffboard">
           <property name="text">
            <string>Offboard</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="autoRepeat">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutPath">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Path:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="path"/>
         </item>
         <item>
          <widget class="QPushButton" name="pathButton">
           <property name="text">
            <string>Choose File</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutGps">
       <item>
        <widget class="QLabel" name="labSelectGps">
         <property name="text">
          <string>Select GPS File:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="GPSComboBox"/>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayoutDest">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutDest">
         <item>
          <widget class="QLabel" name="labDestination">
           <property name="text">
            <string>Destination:</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacerRTS">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="RTScheckBox">
           <property name="text">
            <string>RTS</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayoutCoordinate">
         <item>
          <widget class="QLabel" name="labLat">
           <property name="text">
            <string>lat:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="latLineEdit">
           <property name="text">
            <string>lat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="labLon">
           <property name="text">
            <string>lon:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lonLineEdit">
           <property name="text">
            <string>lonasdasdasdasd</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="ModeSelector">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Ok</set>
       </property>
       <property name="centerButtons">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
